client/components/Players/ShakaPlayer.vue

```diff
@@ -39,19 +38,37 @@ export default Vue.extend({
       source: '',
       // eslint-disable-next-line @typescript-eslint/no-explicit-any
       player: null as any,
-      // eslint-disable-next-line @typescript-eslint/no-explicit-any
-      ui: null as any,
       // eslint-disable-next-line @typescript-eslint/no-empty-function
       unsubscribe(): void {}
     };
   },
   computed: {
-    ...mapGetters('playbackManager', ['getCurrentItem']),
-    ...mapState('playbackManager', ['currentTime', 'lastProgressUpdate']),
+    ...mapGetters('playbackManager', [
+      'getCurrentItem',
+      'getCurrentlyPlayingMediaType'
+    ]),
+    ...mapState('playbackManager', [
+      'lastProgressUpdate',
+      'currentTime',
+      'currentVolume'
+    ]),
     ...mapState('deviceProfile', ['deviceId']),
     ...mapState('user', ['accessToken']),
-    poster(): ImageUrlInfo {
-      return this.getImageInfo(this.getCurrentItem, { preferBackdrop: true });
+    poster(): ImageUrlInfo | string {
+      if (this.getCurrentlyPlayingMediaType === 'Video') {
+        return this.getImageInfo(this.getCurrentItem, { preferBackdrop: true });
+      } else {
+        return '';
+      }
+    },
+    mediaElement(): string {
+      if (this.getCurrentlyPlayingMediaType === 'Audio') {
```

Comment:
No way we can make this an enum?

Reply:
It'd have to be done server-side, since this essentially just gives you the media type from the currently playing item.